tests/test_outliers/test_winsorizer.py

```diff
@@ -164,6 +164,46 @@ def test_quantile_capping_both_tails_with_fold_15_percent(df_normal_dist):
     assert np.round(df_normal_dist["var"].max(), 3) > np.round(0.11823196128033647, 3)
 
 
+def test_indicators_are_added(df_normal_dist):
+    transformer = Winsorizer(
+        tail="both", capping_method="quantiles", fold=0.1, add_indicators=True)
+    X = transformer.fit_transform(df_normal_dist)
+    assert X.shape[1] == 3 * df_normal_dist.shape[1]
+    assert np.all(X.iloc[:, df_normal_dist.shape[1]:].sum(axis=0) > 0)
```

Comment:
I am not sure what this is testing. But for maximum security, could we have a dataframe hardcoded with the expected results, and check that the transformer returns a dataframe exactly like the one we expect?

I think we have some examples where we use pandas assert dataframe equals in this file that you could borrow. 

This way, we are sure that the left indicators are used for values on the left tail and the right indicators for values on the right tail.